expr/funcs.go

```diff
@@ -2,55 +2,16 @@ package expr
 
 import "github.com/raintank/metrictank/api/models"
 
-//go:generate stringer -type=argType
-type argType uint8
-
-// argument types. to let functions describe their inputs and outputs
-// potentially not as strict as reality (e.g. movingAverage windowsize is categorized as a str) that's why we have the extra validation step
-const (
-	series      argType = iota // a single series
-	seriesList                 // a list of series
-	seriesLists                // one or multiple seriesLists
-	integer                    // number without decimals
-	integers                   // one or multiple numbers without decimals
-	float                      // number potentially with decimals
-	str                        // string
-	boolean                    // True or False
-)
-
-type optArg struct {
-	key string
-	val argType
-}
-
 type Func interface {
```

Comment:
this entire library is a library for graphite processing. So until we introduce distinct sets of functions for graphite processing vs other language/system,  I think we can leave the "Graphite" part implied. following your logic we'd need to add "Graphite" to a lot of other types and identifiers as well. e.g. Series -> GraphiteSeries, which I think we should not do.